.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+name: Python Wheels
+
+on: [push, pull_request]
+
+concurrency:
+  group: release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_py36_plus_wheels:
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        buildplat: ["win_amd64", "win32"]
+        python: ["cp36", "cp37", "cp38", "cp39", "cp310"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.4.0
+        env:
+          CIBW_BUILD: ${{ matrix.python }}-${{ matrix.buildplat }}
+          CIBW_TEST_COMMAND: "python -c \"import winkerberos;print(winkerberos.__version__)\""
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.python }}-${{ matrix.buildplat }}
+          path: ./wheelhouse/*.whl
+
+  build_py35_wheels:
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        buildplat: ["win_amd64", "win32"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v1.12.0
+        env:
+          CIBW_BUILD: "cp35-${{ matrix.buildplat }}"
+          CIBW_TEST_COMMAND: "python -c \"import winkerberos;print(winkerberos.__version__)\""
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: cp27-win_amd64
+          path: ./wheelhouse/*.whl
+
+  make_sdist:
+    name: Make SDist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build SDist
+        run: |
+          pip install build check-manifest
+          check-manifest -v
+          python -m build --sdist .
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "sdist"
+          path: dist/*.tar.gz
```

Comment:
Since this is Windows I've been doing zip files instead of tar files. Does Windows have native support for tar yet? I don't think Windows 10 does.
https://pypi.org/project/winkerberos/#files

Reply:
We can manually use `python setup.py sdist --formats=zip` for this release, but invoking `setup.py` directly is deprecated, and the `python -m build sdist` tool doesn't support zip files.  It is expected that you're installing with a fronted (like `pip`) that can handle tar files.

Reply:
I've always thought of sdist as a way to distribute the source code, docs, etc. rather than _only_ a way to give setuptools or pip the source if there was no binary egg or wheel available. The docs claim, perhaps incorrectly, that zip is the default on Windows:

https://docs.python.org/3/distutils/sourcedist.html

Reply:
I relent. See this thread:

https://github.com/pypa/build/issues/243

Which points to this text in PEP517:

"Although historically zip files have also been used as sdists, this hook should produce a gzipped tarball. This is already the more common format for sdists, and having a consistent format makes for simpler tooling."